pkg/plugins/verify-owners/verify-owners_test.go

```diff
@@ -832,6 +832,11 @@ var ownersAliases = map[string][]byte{
 	"collaborators": []byte(`aliases:
   foo-reviewers:
   - alice
+`),
+	"emptyAliasGroupOwnersAliases": []byte(`aliases:
```

Comment:
Should we also test here the non-braces version, or does that get caught somewhere else already?